.gitignore

```diff
@@ -18,8 +18,9 @@
 *.log
 *.pot
 *.pyc
-local_settings.py
+settings_local.py
```

Comment:
What is this change for?

Reply:
This change do navigate in project tree more simple, because usualy use alphabetical sort file order. Same way too use for tenplates_local, static_local

Reply:
What with the developers that already have `local_settings.py` in their projects dirs?

Reply:
Saleor have only one link to local_settings, - in .gitignore
we can use any name local_settings file, this don't have big weight